api/resources/hems.py

```diff
@@ -0,0 +1,69 @@
+from flask_restful import Resource
+from flask import request
+from models.schema.hems import hems
+from config import app
+import json
+import hashlib
+import base64
+from tangle import send_to_iota
+
+
+class Hems (Resource):
```

Comment:
The requirement has changed to `bems` & `cems` , not `bems` and `hems` .

Reply:
I think the resolve way is: move `send_dr_event` from hems to bems, and remove hems.
Is this correct?

Reply:
hems->cems
Fix in 0f8e586.